Libraries/test/TestServerlessApp/serverless.template

```diff
@@ -33,36 +33,6 @@
         }
       }
     },
-    "GreeterSayHelloAsync": {
```

Comment:
generator change, I hope it will not shuffle next time. Else we have to fix the logic to not move around functions.